include/attribute-based/bsw/ciphertext-policy/BSWCiphertextPolicyAttributeBasedEncryptionEncryptedMessage.h

```diff
@@ -0,0 +1,26 @@
+
+#ifndef __CRYPTID_CRYPTID_ENCRYPTEDMESSAGE_ABE_H
+#define __CRYPTID_CRYPTID_ENCRYPTEDMESSAGE_ABE_H
+
+#include "gmp.h"
+#include "elliptic/AffinePoint.h"
+#include "attribute-based/bsw/ciphertext-policy/BSWCiphertextPolicyAttributeBasedEncryptionAccessTree.h"
+#include "complex/Complex.h"
+
+typedef struct BSWCiphertextPolicyAttributeBasedEncryptionCtildeSet
+{
+	Complex Ctilde;
```

Comment:
Our naming convention is: start with lowercase first letter and the internal words start with capital letters. so you should use `cTilde` instead of `Ctilde`. Also check for other mistakes like that.